src/odemis/gui/cont/tabs.py

```diff
@@ -4018,6 +4018,14 @@ def _createFocusStreams(self, focuser, hw_settings_config):
         """
         focus_streams = []
         dets = GetSpectrometerFocusingDetectors(focuser)
+
+        # Sort to have the first stream corresponding to the same detector as the
+        # stream in the alignment view. As this stream will be used for showing
+        # the slit line after the autofocus.
+        if self._ccd_stream:
+            ccd = self._ccd_stream.detector
+            dets = sorted(dets, key=lambda d: d.name == ccd.name, reversed=True)
+
```

Comment:
Just a small thing here, it's `reverse` not `reversed` :)

Reply:
So strange, I really had tested the code.... Thanks a lot for catching this!
It's now fixed.